src/main/java/org/cbioportal/cdd/config/CDDAppConfig.java

```diff
@@ -35,21 +34,16 @@
 @EnableCaching
 public class CDDAppConfig {
 
-    @Value("${ehcache.persistence.path}")
-    private String ehcachePersistencePath;
-
-    @Bean
-    public JCacheManagerFactoryBean jCacheManagerFactory() throws Exception {
-        File file = ResourceUtils.getFile("classpath:ehcache.xml");
-        JCacheManagerFactoryBean factory = new JCacheManagerFactoryBean();
-        factory.setCacheManagerUri(file.getAbsoluteFile().toURI());
-        return factory;
+    @Bean(destroyMethod = "close")
```

Comment:
The close destroyMethod might be called by default, so we probably do not need to specify it here.

Reply:
I think it is called by default but I don't think it hurts to keep this here